examples/chain.js

```diff
@@ -1,9 +1,9 @@
-var Nemo = require("../");
-var config = {
-  "driver": {
-    "browser": "phantomjs"
-  },
-  'data': {
+const Nemo = require('../');
+const config = {
+  'driver': {
+    'browser': 'phantomjs'
+  }
+  ,'data': {
```

Comment:
I was going to make the same comment.  It's a small thing, but it jumped out at me as unnecessary.